core-bundle/src/Resources/contao/drivers/DC_Table.php

```diff
@@ -1594,7 +1609,12 @@ public function deleteChilds($table, $id, &$delete)
 			{
 				foreach ($objDelete->fetchAllAssoc() as $row)
 				{
-					$this->denyAccessUnlessGranted(ContaoCorePermissions::DC_ACTION_DELETE, new DataContainerSubject($v, $row['id']));
+					$this->loadActiveRecord($row['id']);
```

Comment:
we should rather fetch the full record when getting delete IDs instead of re-querying every row again

Reply:
Improved in e95e7478fe02f10a0b59081914cdefb514259326.